crates/matrix-sdk/src/test_utils/mocks.rs

```diff
@@ -432,6 +487,18 @@ impl MatrixMockServer {
         }
     }
 
+    /// Create a prebuild mock for paginating room message with the `/messages`
+    /// endpoint.
+    pub fn mock_room_messages(&self, limit: Option<u32>) -> MockEndpoint<'_, RoomMessagesEndpoint> {
```

Comment:
Can `limit` be set optionally on `MockEndpoint<'_, RoomMessagesEndpoint>`, since this is basically a builder pattern?